src/RNS3.js

```diff
@@ -32,7 +32,7 @@ export class RNS3 {
       contentType: file.type
     });
 
-    let url = `https://${ options.bucket }.s3.amazonaws.com`;
+    let url = `https://${ options.bucket }.${options.url || 's3.amazonaws.com'}`;
```

Comment:
This needs to be `awsURL`, no?

Also, I go back and forth all the time on whether I think something like url should be all caps or just the u when it comes after something else in a variable name.

`awsUrl` vs `awsURL`

I think I'm leaning towards `awsUrl` but honestly don't care that much.